src/Stripe.net/Services/FileLinks/StripeFileLinkService.cs

```diff
@@ -0,0 +1,79 @@
+namespace Stripe
+{
+    using System.Collections.Generic;
+    using System.Net;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Stripe.Infrastructure;
+
+    public class StripeFileLinkService : StripeService
+    {
+        private static string classUrl = Urls.BaseUrl + "/file_links";
+
+        public StripeFileLinkService()
+            : base(null)
+        {
+        }
+
+        public StripeFileLinkService(string apiKey)
+            : base(apiKey)
+        {
+        }
+
+        public virtual StripeFileLink Create(StripeFileLinkCreateOptions createOptions, StripeRequestOptions requestOptions = null)
+        {
+            return CreateAsync(createOptions, requestOptions).Result;
+        }
+
+        public virtual async Task<StripeFileLink> CreateAsync(StripeFileLinkCreateOptions createOptions, StripeRequestOptions requestOptions = null, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            return Mapper<StripeFileLink>.MapFromJson(
+                await Requestor.PostStringAsync(
+                    this.ApplyAllParameters(createOptions, classUrl, false),
+                    this.SetupRequestOptions(requestOptions),
+                    cancellationToken).ConfigureAwait(false));
```

Comment:
I would format the last line (ConfigureAwait) like so:

```
return Mapper<StripeFileLink>.MapFromJson(
                 await Requestor.PostStringAsync(
                     this.ApplyAllParameters(createOptions, classUrl, false),
                     this.SetupRequestOptions(requestOptions),
                     cancellationToken
                 ).ConfigureAwait(false));
```

I like this JavaScript/jQuery style formatting practice. A quick glance shows where the function call ends imo

Reply:
@jaymedavis We now use StyleCop for the coding styles and one of the rules requires that the last bit is on the same line as cancelation token so it would have to look like what @ob-stripe did.